src/modules/balance/ModuleBalance.vue

```diff
@@ -29,7 +28,7 @@
       icon-align="left"
     >
       <template #rightHeaderContainer>
-        <div class="d-flex align-center">
+        <div class="d-flex align-center ml-8 mt-3 mt-sm-0">
```

Comment:
Right now on XS button send tx looks a bit odd, can you please make sure Eth Price change and price stay on 1 line and button on the other and its is centered? 
<img width="529" alt="Screen Shot 2021-06-29 at 1 11 31 PM" src="https://user-images.githubusercontent.com/16910687/123861184-bcafb000-d8db-11eb-9a25-228db6719610.png">
<img width="428" alt="Screen Shot 2021-06-29 at 1 11 27 PM" src="https://user-images.githubusercontent.com/16910687/123861187-bde0dd00-d8db-11eb-870d-562006796659.png">
<img width="311" alt="Screen Shot 2021-06-29 at 1 11 01 PM" src="https://user-images.githubusercontent.com/16910687/123861192-be797380-d8db-11eb-8d10-871c536cdfc6.png">

Reply:
Fixed

![Screenshot from 2021-07-02 18-31-06](https://user-images.githubusercontent.com/9893774/124339522-e0852700-db63-11eb-9275-3c59a753bd56.png)
![Screenshot from 2021-07-02 18-32-09](https://user-images.githubusercontent.com/9893774/124339523-e11dbd80-db63-11eb-9266-ff1d7696e8ac.png)
![Screenshot from 2021-07-02 18-32-16](https://user-images.githubusercontent.com/9893774/124339525-e1b65400-db63-11eb-8302-8814b1ca66ee.png)